uk_geo_utils/management/commands/update_onspd.py

```diff
@@ -0,0 +1,27 @@
+from uk_geo_utils.base_updater import BaseUpdater
+from uk_geo_utils.helpers import get_onspd_model
+from uk_geo_utils.management.commands.import_onspd import (
+    HEADERS,
+)
+from uk_geo_utils.management.commands.import_onspd import (
+    Command as LocalImporter,
+)
```

Comment:
When I wrote the original version of this in EE, I imported the whole management command as a class and just called a method on it because it was code in a package and I couldn't easily reach in and change it from EE. The whole `LocalImporter` thing was a hack really.
That feels like a bit of an odd thing to be doing now.
I think I'd be inclined to rewrite the `import_x` commands such that they extend `BaseUpdater` rather than writing commands that extend `BaseUpdater` and then import another management command to use some of it. Particularly given `import_x` and `update_x` should be basically identical in terms of the end result (obviously the internals are different). Do you think there is a strong reason to keep the old `import_x` variants around?

Reply:
Done in: 13deba20c14a and 9c2f49e3d1d